core/designsystem/src/main/java/co/kr/tnt/designsystem/component/TextField.kt

```diff
@@ -291,14 +292,24 @@ fun TnTOutlinedTextField(
                 keyboardType = keyboardType,
             ),
         )
-        Text(
-            text = "${value.length}/$maxLength",
-            style = TnTTheme.typography.label2Medium,
-            color = counterColor,
-            modifier = Modifier
-                .padding(vertical = 8.dp)
-                .align(Alignment.End),
-        )
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier.padding(vertical = 8.dp),
+        ) {
+            if (isError && !warningMessage.isNullOrEmpty()) {
```

Comment:
```suggestion
            if (isError && warningMessage.isNullOrEmpty().not()) {
```

요런 방식으로도 쓸 수 있어염 ㅎ.ㅎ 근데 이건 취향차이 !